### PR TITLE
fix [object Object] on client basic and software panel. issue 437

### PR DESCRIPTION
**Pull request by syntrydy**
fix [object Object] on client basic and software panel.

### CODE REVIEW

File: admin-ui/app/utils/Util.js [hunk 1]
```diff
@@ -14,3 +14,7 @@ export function getNewColor() {
   }
   return color
 }
+
+export function isEmptyObject(obj){
```
Comment (harryandriyan) [inline admin-ui/app/utils/Util.js:18]: I believe no need to create new function, we can use `isEmpty` from `lodash`
```JS
import isEmpty from 'lodash/isEmpty'
```

**Review by harryandriyan — CHANGES_REQUESTED** [review]
Please check my comment, thanks!

File: admin-ui/app/utils/Util.js (hunk 1 above)
Comment (syntrydy) [reply admin-ui/app/utils/Util.js:18]: Yeah that's right let me refactor that. Thanks!

**Review by harryandriyan — APPROVED** [review]
LGTM

**Pull request merged**